packages/luna/src/components/Toast/RadixToast.tsx

```diff
@@ -0,0 +1,77 @@
+import { createContext, useContext, useState } from "react";
+import * as Toast from "@radix-ui/react-toast";
+import cn from "classnames";
+
+type ToastType = "success" | "failed";
+
+type ToastContextProps = {
+  children?: React.ReactNode;
+};
+
+type ToastContextTypes = {
+  open: boolean;
+  type: ToastType;
+  title: string;
+  setType: (type: ToastType) => void;
+  setOpen: (open: boolean) => void;
+  setTitle: (title: string) => void;
+};
+
+export const ToastContext = createContext<ToastContextTypes | null>(null);
+
+export const ToastProvider = ({ children }: ToastContextProps) => {
+  const [open, setOpen] = useState(false);
+  const [title, setTitle] = useState("");
+  const [type, setType] = useState<ToastType>("success");
+  return (
+    <ToastContext.Provider
+      value={{ open, type, title, setOpen, setTitle, setType }}
```

Comment:
Instead of Exposing all of these values, Let's think of this differently.
Every Component will need to just open a toast. 
So how about we make a function that opens a toast and can accept params like `type, title, description`.
Then This will be exposed afterward in other components
like so:
```
const showToaster = (type, title, description?)=>{
setOpen(true)
setTitle(title)
setDescription(description)
}
```

Then we can use it in different components like:
```
const toaster = useToast();
toaster.showToast("Success", "Success Title", "success long description")
```